### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by kathihareesh** [learndevopsonline/learndevopsonline#1540] [Q&A]
How to track the history of a particular user in Linux ?

**Comment by prabhavith** [comment]
@kathihareesh login to the user and give **history** command

**Reply by kathihareesh** [reply]
Thanks for the quick response 

the root user is Ubuntu
I created a new user name, Hareesh 
Now  I need to log in with my Username and Password but for me, it's showing the root user only after logging in to the root user then I'm able to switch my user instead of that.   I should log in with my user only. not through the root user.

**Reply by prabhavith** [reply]
You can get it cleared in q&a session at 9pm. Raghu(@r-devops ) can better understand your issue and provide solution)

**Reply by kathihareesh** [reply]
okay sure
